examples/nextjs/package.json

```diff
@@ -9,6 +9,7 @@
         "lint": "next lint"
     },
     "dependencies": {
+        "@tinymce/tinymce-react": "^5.1.1",
```

Comment:
Ohhh boy.. users will needs to install this in their project? this is not a good DX

Reply:
I added it to the `package.json`, I can remove it but it will be installed on `npm i` or `yarn` because the `tinyMCE` editor is a dependency for the `DotEditableText`. 

If we don't want to depend on any third-party libraries, we should create our own inline editor and provide the `DotInlineEditor` component as a separate package.

cc: @fmontes

Reply:
I thought the file was  `sdks/sdk-react/package.json`, but it is the `examples/nextjs/package.json` where it is not needed.